src/main/views/lists/tabs/contactTypes/contactTypeContent.njk

```diff
@@ -0,0 +1,49 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% set contactTypeItems = [] %}
+{% for contactType in contactTypes %}
+  {% set deleteConfirmUrl = deleteUrl + contactType.id +"?type=" + contactType.type|urlencode %}
```

Comment:
Is there anyway a person could get the type and decode it? What would be the impact of that? May be a non issue. One to think about

Reply:
No real impact only a string type